### DISCUSSION TITLE
Which error does `compare_models` report?

### DISCUSSION THREAD

**Discussion by OleksiyAnokhin** [pycaret/pycaret#2070] [Q&A]
Hi team, a quick question.
I also use PyCaret for rapid modelling, which gives me hints about models to select and apply in `sklearn`.
One of the question I always have. Which metrics are reported in `compare_models` - for train or test?
For instance, if I get RMSE, which is it for train or test? How to get both? I want to understand if the model overfits or not. 
I googled answers but did not find. Appreciate some direction here. Thanks!

**Comment by ngupta23** [comment]
`compare_models` always reports the mean cross-validation scores across the various folds. Cross-validation is only done on the "train" split. Hope that helps.
